import-service/src/functions/importFileParser/handler.ts

```diff
@@ -0,0 +1,85 @@
+import { S3Event } from 'aws-lambda';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { headers } from '@libs/headers';
+import { middyfy } from '@libs/lambda';
+
+import * as AWS from 'aws-sdk';
+import csv from 'csv-parser';
+
+export const importFileParser = async(event: S3Event) => {
+    const s3 = new AWS.S3({ region: 'eu-central-1' });
+    const sqs = new AWS.SQS();
+    const BUCKET_NAME = process.env.BUCKET_NAME;
```

Comment:
suggestion: to use an aggregation approach while storing refs of another class and shift process.env variables in constants not use them directly